aries-core/src/js/components/helpers/Anchors/NavLink.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Anchor } from 'grommet';
+import { Anchor, Text } from 'grommet';
 
-export const NavLink = ({ size, ...rest }) => {
-  return <Anchor navItem size={size} {...rest} />;
+export const NavLink = ({ active, children, label, size, ...rest }) => {
```

Comment:
I don't think you need to pass `children` as a prop while applying `...rest`. That means it can be removed from the PropTypes as well.

Reply:
Good catch. children was leftover from previous approach. Fixing.

Reply:
We are actually using `children` in the `label` prop, so I think it needs to remain, correct?

Reply:
Correct, let's keep it then, thanks!